packages/ember-routing/lib/system/generate_controller.js

```diff
@@ -17,7 +17,7 @@ import {
 */
 
 export function generateControllerFactory(owner, controllerName, context) {
-  let Factory = owner._lookupFactory('controller:basic').extend({
+  let Factory = owner.factoryFor('controller:basic').class.extend({
```

Comment:
if previously, injections where set on `controller:basic` they would now be lost (as we bypass DI if we do `.class`